types/immutability-helper/immutability-helper-tests.ts

```diff
@@ -1,48 +1,37 @@
 import * as update from 'immutability-helper';
 import { newContext } from 'immutability-helper';
 
-namespace TestObjectUpdate {
-  update({}, {
-    foo: {
-      bar: { $set: 'baz' }
-    }
-  });
-}
-
-namespace TestArrayUpdate {
-  update([], {
-    foo: {
-      bar: { $set: 'baz' }
-    }
-  });
-}
-
 namespace TestExtend {
   update.extend('$command', (specValue, originalValue) => originalValue);
 }
 
 namespace TestNewContext {
   update.newContext().extend('$command', (specValue, originalValue) => originalValue);
   newContext().extend('$command', (specValue, originalValue) => originalValue);
-
-  // This shouldn't compile, but we can't test negatives.
-  // newContext().newContext();
 }
 
 namespace TestFromReactDocs {
-  // These are copied from https://facebook.github.io/react/docs/update.html
```

Comment:
I think these tests are probably still useful and should be kept around.

Reply:
They don't test anything type related. immutability-helper has it's own test.

Reply:
This _is_ the immutability-helper test suite, so I'm not sure what you mean by "[it] has its own test". It's a drop-in replacement for `react-addons-update`, so it seems reasonable to include the basic docs examples as a smoke test in addition to any more-thorough type-heavy tests. I don't think these tests are extraneous.